app/features/settings/components/SettingsDrawer.js

```diff
@@ -63,6 +74,24 @@ class SettingsDrawer extends Component<Props, *> {
         this._onNameFormSubmit = this._onNameFormSubmit.bind(this);
     }
 
+    /**
+     * Start Onboarding once component is mounted.
+     *
+     * NOTE: It autonatically checks if the onboarding is shown or not.
+     *
+     * @returns {void}
+     */
+    componentDidUpdate() {
+        if (this.props.isOpen) {
```

Comment:
can you check if `!prevProps.isOpen` here? (just playing safe)

Reply:
Done